tests/functional_tests/models/minimax_m2/test_minimax_m2_conversion.py

```
@@ -0,0 +1,186 @@
+
+        result = subprocess.run(
+            cmd,
+            capture_output=True,
+            text=True,
+            cwd=Path(__file__).parent.parent.parent.parent,
+        )
+
+        if result.returncode != 0:
+            print(f"STDOUT: {result.stdout}")
+            print(f"STDERR: {result.stderr}")
+            assert False, f"MiniMax-M2 {test_name} conversion failed with return code {result.returncode}"
```

Ruff (0.15.2)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)